### PR TITLE
[ALOY-1584] Android: Ti.UI.Toolbar has no click event

### PR DESCRIPTION
**Pull request by ypbnv**
**JIRA:** https://jira.appcelerator.org/browse/ALOY-1584

**Description:**
'onHomeIconItemsSelected ' was not properly inherited from ActionBar when used from Toolbar.
Properties of ActionBar shared between both components are now extracted as constants to avoid future mismatching.

**Test case:**
In JIRA ticket.

### CODE REVIEW

File: Alloy/commands/compile/parsers/Ti.UI.Toolbar.js [hunk 1]
```diff
@@ -16,7 +16,7 @@ exports.parse = function(node, state) {
 		code = '',
 		xmlStyles = {};
 	// Properties inherited from ActionBar when Toolbar is passed to the Activity
-	var inheritedProperties = ['backgroundImage', 'displayHomeAsUp', 'homeButtonEnabled', 'navigationMode', 'onHomeItemSelected'];
+	var inheritedProperties = [CONST.BACKGROUND_IMAGE, CONST.DISPLAY_HOME_AS_UP, CONST.HOME_BUTTON_ENABLED, CONST.NAVIGATION_MODE, CONST.ON_HOME_ICON_ITEM_SELECTED];
```
Comment (feons) [inline Alloy/commands/compile/parsers/Ti.UI.Toolbar.js]: Please define `CONST`.

**Comment by feons** [comment]
@ypbnv, please add a test app for this.

**Comment by ypbnv** [comment]
@feons Updated PR. I am not sure if these two files would be sufficient as test app, so if something else is needed - let me know.

**Comment by feons** [comment]
@ypbnv, in your Alloy repo folder you can use the command  `node tools/create_generated_code.js ALOY-1584` , it'll create `_generated` code folder under `test/apps/testing/ALOY-1584/`, which will be used to confirm by future jake tests that the generated code matches expected code.

**Comment by feons** [comment]
@ypbnv, please move {{test/apps/ALOY-1584/_generated}} to {{test/apps/testing/ALOY-1584/}} and remove {{test/apps/ALOY-1584}}.

**Comment by feons** [comment]
*Tests passed locally.* **Approved.**

**Pull request merged**